### ISSUE TITLE
use existing folder structure

### ISSUE THREAD

**Issue by kaidoloor** [cleidigh/Localfolder-TB#31]
Hi,

thanks for maintaining this. This is a very much needed piece of functionality, especially for corporates (=support for TB). I was active TB user for corporate environment years ago, where we mapped all users' specific subfolder of LocalFolders into a single point on Linux file share, to create a single shared e-mail storage for all users. Finally (after many GB), this started to slow startup and accidential folder move with mouse became an issue. Crazy times.

Now, today, most users have an existing folder structure for their companies, clients, suppliers, hobbies, investments, etc. These are not *.sbd folders, but normally without any extensions. Can LocalFolders be instructed to use these "normal folder trees" as TB Local Folders structure? This would enable co-living of file folders with e-mail folders, and would not require users to duplicate file folder systems in TB local folders. I suspect that *.sbd probably is required by some e-mail storage standard, but maybe you can create an exception here?

Thanks,
Kaido

**Comment by LorenAmelang** [comment]
Yes! I've used EverDesk for many years, storing .eml files in regular Windows folders, but it is apparently abandoned now. It would be so great to be able to just show all those folders in the Thunderbird tree and drag messages to them or open messages from them! 

It worked once - before I installed Localfolder. I added a "test" Local Folder, closed TB, copied some EverDesk .eml files into its "cur" subfolder, and when I restarted TB it saw them! (Only the .eml ones, not .jpg attachments...  But better than nothing.) I've tried to copy more but had no luck at all. 

But I'm not sure I'm using the extension correctly. I don't see any TB interface change from before I installed it. I have TB 78.4.3, with: 
Version
3.0.0-b4
Last Updated
November 11, 2020

What could I be doing wrong?
